readme.md

```diff
@@ -0,0 +1,17 @@
+## How to install Fabric3 on Red Hat distribution ##
+
+1. First, be sure that Python 3.4+ is installed on your computer.
+2. Run "sudo python3 -m pip install Fabric3"
+3. Once it's done, be sure it's well installed by simply running "fab -V"
+
+## How to run Fabric3 ##
+
+1. Clone the repo https://github.com/CSester/automatic-deployment
+2. Create a new folder examples/ in the root directory
```

Comment:
The examples folder including the config.json file should already be in the repository. Please add necessary files

User should only edit tmpFolder (or some default value may be used ??)